e2e/impl/validation.go

```diff
@@ -184,10 +185,38 @@ func (ctx *ValidationContext) MyNodeShouldBeNotBeAValidator() error {
 	return nil
 }
 
+func (ctx *ValidationContext) MyNodeShouldBeAValidator() error {
+	res := &cluster.ExecResponse{}
+	if err := ctx.globalCtx.execCommand(ctx.nodeID(), isValidatingCommand(), res); err != nil {
+		return err
+	}
+	if strings.TrimSpace(res.StdOut) != "true" {
+		log.Debug(res.StdOut)
+		return errors.New("validator is not running")
```

Comment:
Can this return `rest.StdOut`? I know it will usually have the value of `"false"`, bit if it doesn't it would be useful to know what it says.

Reply:
Directly? or more like  errors.Errorf("validator is not running: %s", res.StdOut)

Reply:
More like returning the error